pkg/controllers/ippool/ippool_controller.go

```diff
@@ -114,6 +116,8 @@ func (r *IPPoolReconciler) setReadyStatusTrue(ctx *context.Context, ippool *v1al
 }
 
 func (r *IPPoolReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	// Use once.Do to ensure gc is called only once
+	once.Do(func() { go r.IPPoolGarbageCollector(make(chan bool), servicecommon.GCInterval) })
```

Comment:
once.Do can only ensure GC is called once, but once pod is switched from master to standby, this go routing will be stopped?

Reply:
refer https://github.com/vmware-tanzu/nsx-operator/pull/601